utils/mock-data-loader/banks/index.js

```diff
@@ -157,7 +157,8 @@ const BANKS = [
       { startMonth: 9, endMonth: 11 },
       { startMonth: 12, endMonth: 2 },
     ],
-    isVisibleInTfmUtilisationReports: true,
+    // TODO: Set this to `true` once quarterly reporting is implemented
+    isVisibleInTfmUtilisationReports: false,
```

Comment:
out of interest do we have a pr for this

Reply:
No PR yet, my understanding is this will be have to be reverted in `R2`.